clients/web/src/collections/Collection.js

```diff
@@ -70,6 +107,11 @@ var Collection = Backbone.Collection.extend({
      * result when displaying it to the user.
      */
     pageNum: function () {
+        if (this.filterFunc) {
+            if (this.append) { return 0; }
+            this.pageOffsetStack = this.pageOffsetStack || [];
+            return this.pageOffsetStack.length - 1;
```

Comment:
If this is called before `this.pageOffsetStack`, the `pageNum` returned will be `-1`.  Shouldn't it be `0`?